applications/IgaApplication/custom_utilities/node_curve_geometry_3d.h

```diff
@@ -93,37 +93,37 @@ class NodeCurveGeometry3D
 
     /** Gets the location of the Kratos node at a given index.
      * 
-     * \param Index Index of the node
+     * @param Index Index of the node
      * 
      * @return Location of the Kratos node at the given index.
      */
     VectorType Pole(
         const int Index) const override
     {
-        const auto& node = *Node(Index);
+        const Kratos::Node<3>& node = *Node(Index);
  
         VectorType pole;
-        pole[0] = node[0];
-        pole[1] = node[1];
-        pole[2] = node[2];
+        for (size_t i = 0; i < 3; i++) {
+            pole[i] = node[i];
+        }
  
         return pole;
     }
 
     /** Sets the location of the Kratos node at a given index.
      * 
-     * \param Index Index of the node
-     * \param Value New location of the Kratos node
+     * @param Index Index of the node
+     * @param Value New location of the Kratos node
      */
     void SetPole(
         const int Index,
         const VectorType& Value) override
     {
-        auto& node = *Node(Index);
- 
-        node[0] = Value[0];
-        node[1] = Value[1];
-        node[2] = Value[2];
+        Kratos::Node<3>& node = *Node(Index);
+
+        for (size_t i = 0; i < 3; i++) {
```

Comment:
Please use std::
(And ++i)

Reply:
Yes std, or you wil use c size_t, and you will have problems with pybind or something

Reply:
`++i`/`i++`: I dont see a difference
`size_t`: I forgot… Just to know: what is the exact problem?

Reply:
++i is good practice, even if in this case it is not a difference

I don't know the exact problem with the pybind `size_t` but as far as I rememeber there were some problems in win